tests/Test/HttpClient/Plugin/MockOauth.php

```diff
@@ -49,13 +49,13 @@ public function __construct(
         \Apigee\Edge\HttpClient\Plugin\Authentication\OauthTokenStorageInterface $token_storage,
         HttpClient $httpClient = null,
         JournalInterface $journal = null,
-        ?string $mfa_token = null,
+        ?string $mfaToken = null,
```

Comment:
Keep the snake cased parameter names.

Reply:
Symfony coding standard says parameters should be camelCase for [arguments](https://symfony.com/doc/current/contributing/code/standards.html#naming-conventions).